src/gcp/firestore.ts

```diff
@@ -2,7 +2,13 @@ import { firestoreOrigin } from "../api";
 import { Client } from "../apiv2";
 import { logger } from "../logger";
 
-const apiClient = new Client({
+const prodOnlyClient = new Client({
+  auth: true,
+  apiVersion: "v1",
+  urlPrefix: firestoreOrigin,
+});
+
+const emuOrProdClient = new Client({
   auth: true,
   apiVersion: "v1",
   urlPrefix: firestoreOrigin,
```

Comment:
Was this supposed to be `firestoreOriginOrEmulator`?

Reply:
🤦 Thanks for catching this, I clearly rushed this one